src/tests/utils/convert.js

```diff
@@ -41,6 +41,157 @@ const appDefinition = {
+          params: {
+            state: '{{bundle.inputData.state}}',
+            redirect_uri: '{{bundle.inputData.redirect_uri}}',
+            response_type: 'code',
+            client_id: '{{process.env.CLIENT_ID}}'
+          },
+          method: 'GET'
+        },
+        refreshAccessToken: {
+          body: {
+            grant_type: 'refresh_token',
+            refresh_token: '{{bundle.authData.refresh_token}}'
+          },
+          headers: {
+            'content-type': 'application/x-www-form-urlencoded',
+            accept: 'application/json'
+          },
+          method: 'POST'
+        },
+        getAccessToken: {
+          body: {
+            redirect_uri: '{{bundle.inputData.redirect_uri}}',
+            client_secret: '{{process.env.CLIENT_SECRET}}',
+            code: '{{bundle.inputData.code}}',
+            client_id: '{{process.env.CLIENT_ID}}',
+            grant_type: 'authorization_code'
+          },
+          url: 'https://api.wistia.com/oauth/token',
+          removeMissingValuesFrom: {},
+          headers: {
+            'content-type': 'application/x-www-form-urlencoded',
+            accept: 'application/json'
+          },
+          params: {},
+          method: 'POST'
+        }
+      },
+      type: 'oauth2',
+      connectionLabel: '{{name}}'
+    },
+    version: '1.0.1',
+    triggers: {
+      project: {
+        operation: {
+          perform: {
+            body: {},
+            url: 'https://api.wistia.com/v1/projects.json',
+            removeMissingValuesFrom: {},
+            headers: {
+              Authorization: 'Bearer {{bundle.authData.access_token}}',
+              Accept: 'application/json'
+            },
+            params: { sort_direction: '0', sort_by: 'created' },
+            method: 'GET'
+          }
+        },
+        noun: 'Project',
+        display: {
+          directions: 'this is help text, where does it go?',
+          hidden: false,
+          important: true,
+          description: 'Triggers on a new project created',
+          label: 'New Project'
+        },
+        key: 'project'
+      },
+      codemode: {
+        operation: {
+          perform: {
+            source:
+              "const options = {\n  url: 'https://jsonplaceholder.typicode.com/posts',\n  method: 'GET',\n  headers: {\n    'Accept': 'application/json'\n  },\n  params: {\n    '_limit': '3'\n  }\n}\n\nreturn z.request(options)\n  .then((response) => {\n    response.throwForStatus();\n    const results = z.JSON.parse(response.content);\n\n    // You can do any parsing you need for results here before returning them\n\n    return results;\n  });"
+          }
+        },
+        noun: 'Code',
+        display: {
+          hidden: false,
+          important: true,
+          description: "just runs some code, let's go",
+          label: 'New Code Trigger'
+        },
+        key: 'codemode'
+      }
```

Comment:
I think this is actually the preferred way to do it. Since you need to have both the search and create imported, it usually makes more sense to have it in the index. that also matches the example: https://github.com/zapier/zapier-platform-example-app-search-or-create/blob/017bb5ff8c09bfadf980e88ced6d28741e50e7ac/index.js#L27-L39